scripts/skills/perks/perk_rf_sanguinary.nut

```diff
@@ -1,8 +1,9 @@
 this.perk_rf_sanguinary <- ::inherit("scripts/skills/skill", {
 	m = {
-		FatigueCostRefundPercentage = 25,
-		IsHitHandlingComplete = true,
-		WasBleeding = false
+		IsForceEnabled = false,
+		RequiresWeapon = true,
```

Comment:
Why exactly do we need `IsForceEnabled` aswell as `RequiresWeapon`? It seems like they do the same thing.
I would like comments explaining the purpose of both of them.